extensions/src/AWSSDK.Extensions.CloudFront.Signers/AWSSDK.Extensions.CloudFront.Signers.nuspec

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package> 
+  <metadata> 
+    <id>AWSSDK.Extensions.CloudFront.Signers</id>
+    <title>AWSSDK - Extensions for AWS CloudFront</title>
+    <version>4.0.1.0-preview</version>
+    <authors>Amazon Web Services</authors>
+		<description>
+			This package contains extension methods for creating signed URLs for Amazon CloudFront distributions and
+			for creating signed cookies for Amazon CloudFront distributions using canned or custom policies.
+		</description>
+    <language>en-US</language>
+    <license type="expression">Apache-2.0</license>
+    <projectUrl>https://github.com/aws/aws-sdk-net/</projectUrl>
+    <tags>AWS Amazon aws-sdk-v4</tags>
+    <icon>images\AWSLogo.png</icon>
+    <dependencies>
+      <group targetFramework="net472">
+        <dependency id="AWSSDK.Core" version="4.0.1.0-preview" />
+        <dependency id="AWSSDK.CloudFront" version="4.0.1.0-preview" />
+        <dependency id="BouncyCastle.Cryptography" version="2.4.0" />
+      </group>
+      <group targetFramework="netstandard2.0">
+        <dependency id="AWSSDK.Core" version="4.0.1.0-preview" />
+        <dependency id="AWSSDK.CloudFront" version="4.0.1.0-preview" />
+        <dependency id="BouncyCastle.Cryptography" version="2.4.0" />
+      </group>
+    </dependencies>
+
+  </metadata> 
+  <files>
+    <file src="..\..\..\sdk\nuget-content\AWSLogo.png" target="images\" />
+
+    <file src=".\bin\Release\net472\AWSSDK.Extensions.CloudFront.Signers.dll" target="lib\net472" />
+    <file src=".\bin\Release\net472\AWSSDK.Extensions.CloudFront.Signers.pdb" target="lib\net472" />
+
+    <file src=".\bin\Release\netstandard2.0\AWSSDK.Extensions.CloudFront.Signers.dll" target="lib\netstandard2.0"/>
+    <file src=".\bin\Release\netstandard2.0\AWSSDK.Extensions.CloudFront.Signers.pdb" target="lib\netstandard2.0"/>
```

Comment:
You need to add the .NET Core 3.1 and .NET 8 files.

Reply:
will add these